crates/holochain/src/local_network_tests.rs

```diff
@@ -13,21 +23,11 @@ use holochain_wasm_test_utils::TestWasm;
 use holochain_zome_types::ZomeCallResponse;
 use kitsune_p2p::KitsuneP2pConfig;
 use matches::assert_matches;
-use tempdir::TempDir;
-use tracing::debug_span;
-
```

Comment:
removing this space made the crate-prefixed statements mix with the rest. was that intentional?

Reply:
Yes, because some non-crate-prefixed imports had attached themselves to the bottom of the crate-specific block anyway